packages/block-library/src/heading/editor.scss

```diff
@@ -35,13 +35,13 @@
 	}
 
 	// Adjust line spacing for larger headings.
-	h1.editor-rich-text__tinymce,
-	h2.editor-rich-text__tinymce,
-	h3.editor-rich-text__tinymce {
```

Comment:
I love these removals but I think this is also something we should document in the release... as some themes might have relied on these tinymce classNames. Granted that the components classNames should not be considered public API.

Reply:
We should also drop a note to the themes/design team once this lands to check the twenty* themes.

Reply:
Yes. We should really discourage the use of these classes. The only reasonable uses I've seen is to style the placeholder or boundaries differently. Ideally the default styles should work for anything though. Worth looking into at some point.